components/infobox/commons/infobox.lua

```diff
@@ -67,7 +68,13 @@ function Infobox:build(widgets)
 	if self.bottomContent ~= nil then
 		self.root:node(self.bottomContent)
 	end
-	self.root:node(self.adbox)
+
+	local firstInfobox = not Variables.varDefault('notFirstInfobox')
+	if firstInfobox then
+		self.root:node(self.adbox)
+		Variables.varDefine('notFirstInfobox', 'true')
```

Comment:
```suggestion
	local firstInfobox = Variables.varDefault('is_first_infobox', true)
	if firstInfobox then
		self.root:node(self.adbox)
		Variables.varDefine('is_first_infobox', 'false')
```

Reply:
that would always add the adbox, would need the following:
```suggestion
	local firstInfobox = Variables.varDefault('is_first_infobox', true)
	if firstInfobox == true then
		self.root:node(self.adbox)
		Variables.varDefine('is_first_infobox', 'false')
```

![Screenshot 2021-09-07 15 42 05](https://user-images.githubusercontent.com/75081997/132355350-9170a3b5-54f2-47aa-9407-02deccd449e7.png)

Reply:
It always adds an adbox for the first one, yea, and then it doesn't add any further ones. Not seeing the problem here?

Reply:
Ah, strings are seen as true. Sure, make the proposed change. Leaving up my confused comment